src/pages/home/ReportScreen.tsx

```diff
@@ -526,6 +525,10 @@ function ReportScreen({
             isRemovalExpectedForReportType ||
             isClosedTopLevelPolicyRoom
         ) {
+            // If the report isn't focused, navigation to Concierge Chat should be avoided.
```

Comment:
Can you please edit this slightly to explain WHY we shouldn't navigate depending on the focus state

Reply:
We need here to prevent ReportScreen.tsx from handling "leaving room optimisticData" during the transition time to prevent undesired redirect to concierge when user leaves a room from the same device.

I think we can say it like this: 
```javascript
// We shouldn't redirect to Concierge Chat when report screen is not focused, like during the transition time,
// As this can cause undesired redirection to Concierge Chat when user leaves a room from the same device.
```